aat/befta_recent_executions_info.json

```diff
@@ -0,0 +1 @@
+{"lastExecutionTime":"2020-11-06T16:48:10.733","lastExecutionProjectRepo":"origin\tgit@github.com:hmcts/ccd-case-activity-api.git (fetch)","lastExecutionProjectBranch":"RDM-8934-timeBasedReExecutionOfTestData"}
```

Comment:
Please remove this file. This shouldn't be included in version control. This file will be generated and updated in the locality of test suite executions and just serve in their localities. No need to keep it in version control.